packages/react/src/components/link/Link.tsx

```diff
@@ -55,32 +61,29 @@ export type LinkProps = {
    * Style the link as a button
    */
   useButtonStyles?: boolean;
-} & Omit<
-  React.ComponentPropsWithoutRef<'a'>,
-  'target' | 'href' | 'onPointerEnterCapture' | 'onPointerLeaveCapture' | 'aria-label'
->;
+} & Omit<React.ComponentPropsWithoutRef<'a'>, 'target' | 'href' | 'onPointerEnterCapture' | 'onPointerLeaveCapture'>;
```

Comment:
Did you find out why pointer capturing is omitted? It stands out, because haven't seen it in other components.

Reply:
Nope, but didn't touch it :D